test/funcfile-docker-rt-tests/testfiles/func.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+  "fmt"
+  "log"
+)
+
+func main() {
+  fmt.Println("Hello from Fn for func file 'docker' runtime test !")
+  log.Println("Hello from Fn for func file 'docker' runtime test !")
```

Comment:
Isn't this a duplication?

Reply:
That's for experienting which logging method in GO test get logged with or without -v as I noticed when running the existing cli/main-test.go its fmt.Println() doesn't show up in test output ..

Reply:
Right, so just FYI, `log` package prints to STDERR, `fmt` prints to STDOUT.